src/main/java/appeng/core/sync/packets/PacketPatternSlot.java

```diff
@@ -103,18 +144,13 @@ private void writeItem( final IAEItemStack slotItem, final ByteBuf data ) throws
 		else
 		{
 			data.writeBoolean( true );
-			slotItem.writeToPacket( data );
-		}
-	}
-
-	@Override
-	public void serverPacketData( final INetworkInfo manager, final AppEngPacket packet, final EntityPlayer player )
-	{
-		final EntityPlayerMP sender = (EntityPlayerMP) player;
-		if( sender.openContainer instanceof ContainerPatternTerm )
-		{
-			final ContainerPatternTerm patternTerminal = (ContainerPatternTerm) sender.openContainer;
-			patternTerminal.craftOrGetItem( this );
+			try
+			{
+				slotItem.writeToPacket( data );
+			}
+			catch( final IOException e )
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)